app/controllers/api/v1/workflows_controller.rb

```diff
@@ -11,24 +11,6 @@ class Api::V1::WorkflowsController < Api::ApiController
 
   prepend_before_action :require_login, only: [:create, :update, :destroy, :create_classifications_export]
 
-  # temp action to test perf of Panoptes::RestpackSerializer
```

Comment:
I've branched of off remove_prp_serializer_experiment and was going to rebase, probably should have put that in the description.